### PR TITLE
Mapdl stanage

### PR DESCRIPTION
**Pull request by desmfryan**
The long awaited Ansys Mech/mapdl docs for Stanage.

Updated the modules available & sidebar info, removed mentions of ShARC in the referenceinfo.

### CODE REVIEW

File: stanage/software/apps/ansys/mechanical.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+Ansys Mechanical is a finite element analysis (FEA) tool that enables you to analyze complex product architectures and solve difficult mechanical problems.
+You can use Ansys Mechanical to simulate real world behavior of components and sub-systems, and customize it to test design variations quickly and accurately.
+ANSYS Mechanical has interfaces to other pre and post processing packages supplied by ANSYS and can make use of  built in :ref:`MPI <parallel_MPI>` to utilize multiple cross node CPU and can scale to hundreds of cores.
+
+.. include:: /referenceinfo/imports/software/ansys/ansys-license-restrictions.rst
+  
+----------------
+
+.. include:: /referenceinfo/imports/software/ansys/module-load-list-bessemer.rst
```
Comment (Carldkennedy) [inline stanage/software/apps/ansys/mechanical.rst:26]:
```suggestion
.. include:: /referenceinfo/imports/software/ansys/module-load-list-stanage.rst
```

File: referenceinfo/imports/software/ansys/stanage-sidebar.rst [hunk 2]
```diff
@@ -7,6 +7,6 @@
 
 .. sidebar:: ANSYS
 
-   :Versions: 22.2
+   :Versions: 19.2,2022R2,2023R1,2023R2
```
Comment (Carldkennedy) [inline referenceinfo/imports/software/ansys/stanage-sidebar.rst:10]:
```suggestion
   :Versions: 19.2, 2022R2, 2023R1, 2023R2
```

Comment (Carldkennedy) [reply referenceinfo/imports/software/ansys/stanage-sidebar.rst:10]: Better readability

File: stanage/software/apps/ansys/mechanical.rst (hunk 1 above)
Comment (Carldkennedy) [reply stanage/software/apps/ansys/mechanical.rst:26]: Import correct list

File: referenceinfo/imports/software/ansys/module-load-list-stanage.rst [hunk 3]
```diff
@@ -8,4 +8,7 @@ Module loading
 
 After connecting to Stanage (see :ref:`ssh`),  you can start an :ref:`interactive graphical session <submit_interactive_bessemer>` or submit a batch job using ANSYS programs activating them and making them available with one of the module load commands below: ::
```
Comment (Carldkennedy) [inline referenceinfo/imports/software/ansys/module-load-list-stanage.rst:9]:
```suggestion
After connecting to Stanage (see :ref:`ssh`),  you can start an :ref:`interactive session <submit_interactive_stanage>` or submit a batch job using ANSYS programs activating them and making them available with one of the module load commands below: ::
```

Comment (Carldkennedy) [reply referenceinfo/imports/software/ansys/module-load-list-stanage.rst:9]: Graphical session requires openFlight

File: stanage/software/apps/ansys/mechanical.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+Mechanical / MAPDL
+=========================
+
+.. contents::
+    :depth: 3
+
+----------------
+
+Ansys Mechanical is a finite element analysis (FEA) tool that enables you to analyze complex product architectures and solve difficult mechanical problems.
```
Comment (Carldkennedy) [inline stanage/software/apps/ansys/mechanical.rst:18]:
```suggestion
Ansys Mechanical is a finite element analysis (FEA) tool that enables you to analyse complex product architectures and solve difficult mechanical problems.
```

**Review by Carldkennedy — APPROVED** [review]
Added a few correction and suggestions. Otherwise, LGTM.
Will cause some conflicts with #1956

**Review by nikishe — COMMENTED** [review]
Couldn't catch any more errors , so consider approved pending carls changes

**Pull request merged**